mina-archive/templates/deployment.yaml

```diff
@@ -2,6 +2,8 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
+  annotations:
+    prometheus.io/scrape: 'true'
```

Comment:
Can we have an open dictionary instead?
```
      {{- with .Values.podAnnotations }}
      annotations:
        {{- toYaml . | nindent 8 }}
      {{- end }}
  ```

Reply:
Please do not mind current edits, as these are being used to troubleshoot and specific issue. That is, the current state at anytime of this PR is by no means final.